manager/apis/motion/v1alpha1/batchtransfer_webhook.go

```diff
@@ -243,7 +250,7 @@ func validateDataStore(path *field.Path, store *DataStore) []*field.Error {
 		}
 
 		// Validate Kafka schema registry url
-		if len(store.Kafka.SchemaRegistryURL) != 0 {
+		if store.Kafka.SchemaRegistryURL != "" {
```

Comment:
which lint asked this replacement ?

Reply:
gocritic:
`emptyStringTest: replace `len(store.Kafka.DataFormat) != 0` with `store.Kafka.DataFormat != ""``